cupy/cuda/memory.pyx

```diff
@@ -6,6 +6,7 @@ from cupy.cuda import runtime
 
 from cupy.cuda cimport device
 from cupy.cuda cimport runtime
+import six
```

Comment:
Please move import to L4(and create new import group), because six is a third party library.
see https://www.python.org/dev/peps/pep-0008/#imports